make_ocl.sh

```diff
@@ -40,12 +40,14 @@ NUM_ITERATIONS=26 # including warmup below
 NUM_WARMUP=1
 INTEL_PREFETCH_LEVEL=1 # sets -auto-prefetch-level= for OpenCL compilation
 
-HAM_PATH=thirdparty/ham
-CLU_PATH_INCLUDE=thirdparty/CLU.git
-CLU_PATH_LIB=thirdparty/CLU/clu_runtime
+HAM_PATH_INCLUDE=thirdparty/ham/include/
```

Comment:
TODO: everything HAM can be replaced by thirdparty/noma/bmt, which is the updated stand-alone version of the benchmark timer, grep code for includes of the ham header, and adapt all codes